tests/utils/constants.js

```diff
@@ -0,0 +1 @@
+export const URL = 'http://localhost:4000';
```

Comment:
Nit: I think we should get in the... habit... of keeping newlines at the end of our files

Reply:
What does nit mean? And what's the benefit of adding newlines at the end of files?

Reply:
I've been looking it up and I don't really see how it's a big deal for JavaScript files...

Reply:
Please explain

Reply:
https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file 

Mostly for portability, but also fundamentally a file should have a newline at the end of each line, otherwise it is not...a line...

Reply:
So for example, if a person were to clone the repo to a linux box they may experience weird issues with their own tools...

Reply:
Very cool, Daniel. I'll be sure to add a setting to my autoformatter to add a new line at the end of files from now on.